.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,101 @@
+name: Bug Report
+description: Report a bug or issue
+title: "[Bug]: "
+labels: ["bug", "triage"]
+assignees: []
+body:
+  - type: markdown
+    attributes:
+      value: "#### Thank you for reporting this issue. Please provide all the required information to help us understand and resolve the issue."
+  
+  - type: input
+    id: bug-description
+    attributes:
+      label: "Description"
+      description: "Provide a summary of the issue."
+      placeholder: "Example: Adyen Checkout API returns an unexpected error when..."
+    validations:
+      required: true
+
+  - type: textarea
+    id: reproduce-steps
+    attributes:
+      label: "Steps to reproduce"
+      description: "List all necessary steps to reproduce the issue."
+      placeholder: |
+        1. Install the Adyen API'...'
+        2. Create '...'
+        3. Run '...'
+        4. Observer error '...'
+    validations:
+      required: true
+
+  - type: textarea
+    id: expected-behavior
+    attributes:
+      label: "Expected behavior"
+      description: "Describe what you expected to happen."
+      placeholder: "Example: The Adyen Checkout API should return a 200 status with the correct response payload."
+    validations:
+      required: true
+
+  - type: textarea
+    id: actual-behavior
+    attributes:
+      label: "Actual behavior"
+      description: "Describe what actually happened."
+      placeholder: "Example: The API returns a 500 internal server error."
+
+  - type: textarea
+    id: code-snippet
+    attributes:
+      label: "Code snippet or screenshots (if applicable)"
+      description: "Provide relevant code snippets or screenshots to illustrate the issue."
+      placeholder: "```js\n// Your code here\n```"
+
+  - type: input
+    id: adyen-api-version
+    attributes:
+      label: "Adyen Node API Library version"
+      description: "Specify the version of the Adyen API library you're using."
+      placeholder: "Example: 20.1.2"
+    validations:
+      required: true
+
+  - type: dropdown
+    id: operating-system
+    attributes:
+      label: "Operating System"
+      description: "Select your operating system."
+      options:
+        - Windows
+        - macOS
+        - Linux
+        - Other
+    validations:
+      required: true
+
+  - type: input
+    id: node-version
+    attributes:
+      label: "Node.js version"
+      description: "Specify the Node.js version you're using."
+      placeholder: "Example: 16.14.0"
+    validations:
+      required: true
+
+  - type: input
+    id: npm-version
```

Comment:
Seems irrelevant, we could remove this

Reply:
NPM version sometimes might be a good-to-have info to reproduce a bug as some different versions may handle dependencies and script differently. We can instead mark it optional. What do you think?